src/components/endpointCreation/NewCaptureSpecForm.tsx

```diff
@@ -43,6 +48,8 @@ function NewCaptureSpecForm(props: NewCaptureSpecFormProps) {
             </Alert>
         );
     } else if (sourceSchema !== null) {
+        const uiSchema = generateUISchema(sourceSchema);
+
```

Comment:
If we generate this and then pass it into JsonForms (down below) then they will no generate this for us.